Source/Particles/PulsarParameters.cpp

```diff
@@ -0,0 +1,91 @@
+#include <PulsarParameters.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_Print.H>
+#include <AMReX_RealVect.H>
+#include <AMReX_REAL.H>
+#include <AMReX_GpuQualifiers.H>
+
+namespace PulsarParm
+{
+   std::string pulsar_type;
+
+   AMREX_GPU_DEVICE_MANAGED amrex::Real omega_star;
+   AMREX_GPU_DEVICE_MANAGED amrex::Real B_star;
+   AMREX_GPU_DEVICE_MANAGED amrex::Real R_star;
+   AMREX_GPU_DEVICE_MANAGED amrex::Real dR_star;
+   AMREX_GPU_DEVICE_MANAGED int EB_external = 0;
+   AMREX_GPU_DEVICE_MANAGED int E_external_monopole = 0;
+   AMREX_GPU_DEVICE_MANAGED amrex::Vector<amrex::Real> center_star(3,0.0);
```

Comment:
When changing to `amrex::GpuArray` in the header, this line changes to:

```
AMREX_GPU_DEVICE_MANAGED amrex::GpuArray<amrex::Real, AMREX_SPACEDIM> center_star;
```

Reply:
done.